### Theory:
 
 &nbsp;
 
Ribosomes are complex structures found in all living cells which functions in protein synthesis machinery. Basically ribosome’s consists of two subunits, each of which is composed of protein and a type of RNA, known as ribosomal RNA (rRNA).  Prokaryotic ribosomes consist of 30S subunit (small sub unit) and 50S subunit (large sub unit) which together make up the complete 70S ribosome, where S stands for Svedberg unit non-SI unit for sedimentation rate. 30S subunit is composed of 16S ribosomal RNA and 21 polynucleotide chains while 50S subunit is composed of two rRNA species, the 5S and 23S rRNAs. The presence of hyper variable regions in the 16S rRNA gene provides a species specific signature sequence which is useful for bacterial identification process. 16S Ribosomal RNA sequencing is widely used in microbiology studies to identify the diversities in prokaryotic organisms as well as other organisms and thereby studying the phylogenetic relationships between them. The advantages of using ribosomal RNA in molecular techniques are as follows:
 
 &nbsp;
 
1. Ribosomes and ribosomal RNA are present in all cells.
2. RNA genes are highly conserved in nature.
3. Culturing of microbial cells is absent in the sequencing techniques.
  
 &nbsp;
 
Signature sequences are some specific base sequences which are always found in all groups of organisms. These unique DNA sequences are about 5–10 bases long and found specifically in the 16S rRNA location, and are unique to many major groups of prokaryotic organisms, archaea and Eukarya.  The average lengths of the structural rRNA genes are 1,522 bp, 2,971 bp, and 120 bp respectively for 16S, 23S, and 5S rRNAs.
 
 &nbsp;
 
Conventional microbiology techniques such as culturing of microorganisms, biochemical tests and other related methods are used worldwide to identify most of the bacteria, fungi and other pathogens, still it takes about 8 to 20 hours for an accurate result. New diagnostic techniques have been developed to overcome the limitations of conventional microbiological methods for identifying etiological agents of infections. Nucleic acid based detection methods help in the detection of genomic materials and thus many genetic or infectious diseases can now be diagnosed by performing a study of relevant DNA sequence by nucleic acid-based techniques.


### Steps in Ribosomal RNA Sequencing:

 &nbsp;
  
#### Extraction of DNA
 
The genetic material of all living organisms contains information that is crucial for heredity.  The DNA segments that carry this genetic information are called genes which are necessary for genetic analysis, which is used for scientific, medical, or forensic purposes. DNA is not free inside the nucleus of a cell. It is usually associated with different proteins and encased in a cellular membrane. Presence of these proteins, lipids, polysaccharides and some other organic or inorganic compounds in the DNA preparation can interfere with DNA analysis methods. Factors affecting the methods of DNA isolation are the age, source, and size of the sample. The presence of proteins, lipids, polysaccharides etc. during DNA preparation can interfere with DNA analysis methods by reducing the quality of DNA. The extraction methods to efficiently purify DNA from various sources have to be adapted depending on factors such as sample size, the freshness of the sample, and the biochemical content of the cells from which DNA is being extracted. The isolation method must vary depending on the size of sample. In the case of bacteria, the main biochemicals present in a cell extract are protein, DNA and RNA.

 &nbsp;
 
#### Action of Different Chemicals in DNA Extraction
 
1. TE buffer: It is a buffer used for the storage of nucleic acids (DNA and RNA), and also to prevent it from degradation.

2. Lysozyme: Enzyme that is used for degrading the cell wall of the organism.

3. SDS: SDS is strong anionic detergent that helps in solubilizing the proteins and lipids present in the membranes. It exposes the chromosomes that contain the DNA, also helps in releasing DNA from histones and other DNA binding proteins by denaturing them.

4. Proteinase K: Degrades most of the protein impurities in the DNA(Deproteination).

5. Phenol: Helps in removing most of the protein impurities from the DNA.

6. Chloroform: Prevents the shearing effect of DNA during the extraction process.

7. Isoamyl alcohol: Reduce the formation of the foams in extraction techniques.

8. Sodium acetate: Precipitates DNA in the solution

9. Absolute ethanol: Precipitate the DNA out of solution.
 
&nbsp;

### Polymerase Chain Reaction
 
PCR is a rapid, automated technique used for the amplification of specific DNA sequences, invented by Kary B Mullis in 1983, and for which he won the Nobel Prize in Chemistry in 1993. PCR has gained over nucleic acid based detection techniques due to its simplicity, specificity, rapidity and sensitivity. In this technique only the DNA of the organism is examined, not the entire viable microorganism, as a result, the pathogenic microorganism can also be evaluated. Valuable genetic information about the microorganisms can be obtained quickly. PCR has become an essential tool in research laboratories and is also creating an impact in diagnostic laboratories.

#### Steps in the PCR process
  
&nbsp;

A technique for amplification of a specific fragment of DNA of interest by a series of successive cycles. By this process, a single molecule of template DNA can generate over a billion copies of itself after 30 cycles of exponential replication. There are three phases in the cycle, each of which occurs at a different temperature. The phases can be performed in an instrument known as thermocycler, which provides these different temperatures.
 
&nbsp;

1. Denaturation: During this process, the double helical arrangement of the sample DNA (template DNA) is denatured at a temperature of about 94°C- 95°C. The two strands get separated out.
2. Annealing: In this step, the primers, which are the sequences of DNA added to the reaction mixture anneal with the complementary (similar or matching) sequences in the template DNA. This occurs at different temperatures.
3. Extension: This is the final stage of the PCR cycle, occurs at 72°C when the enzyme Polymerase added to the reaction mixture, make the primers extend along the length of the DNA strand.
  
&nbsp;

The enzyme polymerase (mainly Taq Polymerase) was isolated from a thermophilus organism, so that it can function optimally at temperature around 72°C and thus allowing the DNA synthesis step to be performed at higher temperatures.
  
&nbsp;

### Agarose Gel Electrophoresis
 
&nbsp;

Electrophoresis is a technique used in the laboratory for separating charged molecules. DNA is negatively charged and it can be moved through an agarose matrix by means of electric current. Shorter molecules migrate more easily and move faster than longer molecules through the pores of the gel and this process is called sieving. The gel might be used to look at the DNA in order to quantify it or to isolate a particular band. The DNA can be visualized in the gel by the addition of ethidium bromide. It is an intercalating agent which intercalates between nucleic acid bases and allows the convenient detection of DNA fragments in gel. When exposed to UV light, it will fluoresce with an orange color. After the running of DNA through an EtBr-treated gel, any band containing more than ~20 ng DNA becomes distinctly visible under UV light. The migration rate of the linear DNA fragments through agarose gel is proportional to the voltage applied to the system. As voltage increases, the speed of DNA also increases. But voltage should be limited because it heats and finally causes the gel to melt.
 
&nbsp;

### Elution of  DNA
 
Elution describes the extraction of specific bands of DNA from agarose gels in which they are separated through electrophoresis. The first step in extracting DNA is identifying the DNA band which is to extract, by illuminating under UV light.  Recovery of DNA from agarose gels by electrophoresis onto DEAE-cellulose membrane is one of the rapid and effective methods. Electro elution is a rapid method for the successful isolation of DNA especially for larger DNA fragments, where the gel fragment containing the DNA band is cut out of the gel and placed into a dialysis bag with some buffer. The bag is then kept into a gel box, which contains the same buffer, and then subjected to an electric current. The extracted DNA precipitates out from the solution. Low melting point agarose is widely employed for the separation of DNA from agarose. Low melting point agarose melts at a lower temperature than standard agarose since it does not denature DNA structure.
 
&nbsp;

### Radiolabeling Technique

The ability to label nucleic acids is one of the most fundamental tools in molecular biology techniques. Radiolabeling is one of the best methods of choice for the most sensitive when it would be difficult to visualize a nonradioactive label, such as the gel mobility shift assay, where the probe remains within the gel matrix. Radioactive tracers have the ability to detect small quantities of substances of interest. In case of radio labeling 16S ribosomal sequence, the specific sequence is in tiny amount compared to the large genomic size of the organism. The direct measurement methods such as ultraviolet absorption, staining with specific dyes are not applicable in most cases due to the limited sensitivities of the methods. Modern techniques such as autoradiography, phosphor imaging and liquid scintillation counting techniques are recently applied for detecting the radioactive tracers.


### Restriction Digestion

Restriction enzymes are endonucleases which cleave double-stranded DNA at specific oligonucleotide sequences. The specific sites at which they cleave the nucleic acids in order to generate a set of smaller fragments are called restriction sites. The natural function of restriction enzymes in bacteria may be the destruction of foreign DNA that may enter the bacterial cell. But the cells own DNA is not cleaved by these restriction enzymes. This self protection is achieved by the help of the specific DNA methyltransferase enzyme which will methylates the specific DNA sequence for its respective restriction enzymes by transferring methyl groups to adenine or cytosine residues to produce N6-methyladenine or 5-methylcytosine. An interesting feature of restriction endonuclease is that they commonly recognize recognition sequences that are mostly palindromes - they shows the same forward (5' to 3' on the top strand) and backward (5' to 3' on the bottom strand) sequences. The DNA fragments of varying length can be separated by gel electrophoresis and stained with ethidium bromide and can be photographed for future studies.


### Southern Blotting

DNA fragments obtained by restriction digestion and separation on gel can be transferred from the gel by blotting to nitrocellulose or nylon membrane that binds the DNA. The DNA thus bound to the nitrocellulose membrane is converted to the single-stranded forms (denaturation) and then treated with radioactive single-stranded DNA probes. These will hybridize with the homologous DNA, if present in the sample, to form radioactive double stranded segments. Finally the bands are visualized by autoradiography with x-ray film or by phosphor imaging techniques. This highly sensitive technique for identifying DNA fragments by DNA-DNA hybridization is called Southern blotting technique.
 


#### Steps in Southern blotting technique
 

. The DNA to be analyzed is first digested to completion with a restriction enzyme.
. The complex mixture of fragments is subjected to gel electrophoresis
. The restriction fragments present in the gel are denatured with alkali.
. Transfer the gel onto a nitrocellulose filter or nylon membrane by blotting.
. The filter is then incubated under hybridization conditions with a specific radiolabeled DNA probe.
. The probe then hybridizes to the complementary DNA restriction fragment.
. Excess probe is washed away and the probe bound to the filter is detected by autoradiography.
 

### Autoradiography
 

Autoradiography is a technique used to detect radioactive compounds with the aid of photographic emulsion, which is basically a piece of X-ray film. The radioactive DNA fragments on a gel is then placed in contact with X-ray film and kept it in the dark area for few hours, or even days. The radioactive emission from the bands of the DNA exposes the film. When the film is developed dark bands appear corresponding to the DNA bands present in the gel. In other words, the DNA bands in the agarose gel take a picture of themselves, and hence the name autoradiography. To enhance the autoradiography sensitivity, intensifying screen can be used which is coated with a compound that fluoresces when it is excited by β-rays at low temperature. The electrophoresed radioactive DNA fragments are seen as parallel lanes and it position depends on the size of the fragments. But the DNA bands are invisible but the positions can be identified with some dotted lines. This gel is then placed in contact with a piece of X-ray film. Leave it for several hours to days. Finally, develop the film to see where the radioactivity has exposed the film, which shows the position of DNA bands on the gel. The large, slowly migrating bands are thought to be more radioactive, and thus the bands corresponds to them on the autoradiogram are darkest than the others.

 
#### Applications of 16S Ribosomal RNA in Microbiology
 

. 16S rRNA gene sequencing has been established as the “gold standard” for identiﬁcation and taxonomic classiﬁcation of bacterial species.

. Comparison of the bacterial 16S rRNA sequence has been emerged as a valuable genetic technique and can lead to the recognition of novel pathogens such as Mycobacterium species.

. The hyper variable regions of 16S rRNA gene sequences provide species-specific signature sequences useful for bacterial identification.

. In medical microbiology, 16S rRNA sequencing serves as a rapid and cheap alternative to phenotypic methods of bacterial identification.

. It is also capable of reclassifying bacteria into completely new species, or even genera.

. The sequencing techniques can be used to describe new species that have never been successfully cultured in laboratories.
 

In addition to all its applicability, sequencing 16S ribosomal RNA lacks widespread use due to the technical and cost issues. The future work is to translate sequence information from 16S rRNA into suitable biochemical testing process, thereby improving the accuracy and efficiency of genotypic identification phenomenon for better advances even in smaller and routine clinical microbiology laboratories.

